frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -193,23 +193,37 @@ const NavBar = ({ headersData }: Props): ReactElement => {
       </Grid>
     </Grid>
   );
-
   const displayDesktop = (): ReactElement => {
-    return (
-      <Grid container className={toolbar} alignItems="center">
-        <Grid item md={3}>
-          {homeLogo}
-        </Grid>
-        <Grid item md={6} className={search}>
-          {auto()}
+    if (searchBar) {
+      return (
+        <Grid container className={toolbar} alignItems="center">
+          <Grid item md={3}>
+            {homeLogo}
+          </Grid>
+          <Grid item md={6} className={search}>
```

Comment:
You should prevent duplicate code. For this part, you can combine these 2 into 1 then use `<Grid item md={6} className={searchBar ? search : searchHidden}>` which is the only different line.

Reply:
fixed!